### PR TITLE
[Tier-2][Ceph-Volume] Verify zap LVM device

### PR DESCRIPTION
**Pull request by pranavprakash20**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Comment by pranavprakash20** [comment]
run log : https://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-BA3VGP

File: tests/ceph_volume/test_ceph_volume_lvm_tags_after_osd_rm.py [hunk 1]
```diff
@@ -56,16 +67,36 @@ def run(ceph_cluster, **kw):
...
-    osd_tree = loads(out[0])
-    for osd_ in osd_tree.get("nodes"):
-        if osd_.get("id") == osd_id and osd_.get("status") == "up":
-            raise RemoveOsdError("The osd is up after rm operation")
+    # Step to validate lvm zap
+    if config.get("operation"):
+        timeout, interval = 60, 5
+        for w in WaitUntil(timeout=timeout, interval=interval):
+            try:
+                cmd = f" ceph-volume lvm zap --destroy {osd_lvm}"
```
Comment (vamahaja) [inline tests/ceph_volume/test_ceph_volume_lvm_tags_after_osd_rm.py:75]: Can we have cli library for this command ?

**Review by vamahaja — CHANGES_REQUESTED** [review]

File: tests/ceph_volume/test_ceph_volume_lvm_tags_after_osd_rm.py (hunk 1 above)
Comment (pranavprakash20) [reply tests/ceph_volume/test_ceph_volume_lvm_tags_after_osd_rm.py:75]: This command has  to be executed only using  podman and not via cephadm shell. In that case its better to be kept outside of cli? What do you think Vaibhav?

Comment (vamahaja) [reply tests/ceph_volume/test_ceph_volume_lvm_tags_after_osd_rm.py:75]: missed it

**Review by vamahaja — APPROVED** [review]

**Pull request merged**